src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -191,8 +191,6 @@ public override void ViewWillLayoutSubviews()
 			// Use 0 if the NavBar is hidden or will be hidden
 			var toolbarY = NavigationBarHidden || NavigationBar.Translucent || !_hasNavigationBar ? 0 : navBarFrameBottom;
 			toolbar.Frame = new RectangleF(0, toolbarY, View.Frame.Width, toolbar.Frame.Height);
-
-			(Element as IView).Arrange(View.Bounds.ToRectangle());
```

Comment:
We used to have a bunch of code here in XF that would set the Frame on every single view
https://github.com/xamarin/Xamarin.Forms/blob/2f8f4864a4d289dc89a6228e2ca9d6a49993e365/Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs#L187-L200

That was removed thinking that it would no longer be needed, but it looks like it was still important to have something setting the `Frame` for scenarios like this. 

We moved this concept down to the `ParentingViewController` to be a bit less verbose. It seems like that's working fine.